package.json

```diff
@@ -37,10 +37,15 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/react": "^18.3.5",
```

Comment:
These are not needed here, as they belong and are in the nextjs folder package.json.